src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationManageTeamAccess.test.tsx

```diff
@@ -0,0 +1,195 @@
+import PublicationManageTeamAccess from '@admin/pages/publication/components/PublicationManageTeamAccess';
+import { ReleaseSummary } from '@admin/services/releaseService';
+import _userService from '@admin/services/userService';
+import _releasePermissionService, {
+  ContributorInvite,
+  ContributorViewModel,
+} from '@admin/services/releasePermissionService';
+import {
+  render as baseRender,
+  screen,
+  waitFor,
+  within,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router';
+import React, { ReactElement } from 'react';
+
+jest.mock('@admin/services/releasePermissionService');
+const releasePermissionService = _releasePermissionService as jest.Mocked<
+  typeof _releasePermissionService
+>;
+
+jest.mock('@admin/services/userService');
+const userService = _userService as jest.Mocked<typeof _userService>;
+
+describe('PublicationManageTeamAccess', () => {
+  const testPublicationId = 'publication-1';
+
+  const testRelease: ReleaseSummary = {
+    id: 'release-1',
+    timePeriodCoverage: {
+      value: 'AY',
+      label: 'Academic Year',
+    },
+    title: 'Release 1',
+    releaseName: '2000',
+    type: 'AdHocStatistics',
+    publishScheduled: '',
+    latestInternalReleaseNote: 'release1-release-note',
+    approvalStatus: 'Draft',
+    yearTitle: '2000/01',
+    live: false,
+  };
+
+  const testContributors: ContributorViewModel[] = [
+    {
+      userId: 'user-1',
+      userDisplayName: 'User 1',
+      userEmail: 'user1@test.com',
+    },
+  ];
+
+  const testInvites: ContributorInvite[] = [{ email: 'user2@test.com' }];
+
+  test('renders the manage team access table correctly', async () => {
+    releasePermissionService.listReleaseContributors.mockResolvedValue(
+      testContributors,
+    );
+    releasePermissionService.listReleaseContributorInvites.mockResolvedValue(
+      testInvites,
+    );
+    render(
+      <PublicationManageTeamAccess
+        publicationId={testPublicationId}
+        release={testRelease}
+      />,
+    );
+
+    await waitFor(() => {
+      expect(
+        screen.getByRole('heading', {
+          name: 'Release 1 (Not live) Draft',
+        }),
+      ).toBeInTheDocument();
```

Comment:
Don't use `getByRole` selectors (and variants) in `waitFor`. Use faster selectors like `getByText` within the `waitFor` and then perform `getByRole` after it has completed.